src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx

```diff
@@ -14,29 +14,10 @@ redirects:
   - /docs/infrastructure/microsoft-azure-integrations
 ---
 
-Our [Microsoft Azure](https://azure.microsoft.com/) integrations allow you to monitor and report data about your Azure services to New Relic, providing a comprehensive view of your entire architecture in one place. 
+New Relic has several offerings for monitoring your [Microsoft Azure](https://azure.microsoft.com) applications and services. Our Azure offerings include: 
```

Comment:
I think maybe we can make the tone a little friendlier in this intro sentence.

How about something like? "We have several ways for you to monitor your Microsoft Azure apps and services:"